src/Form/Field.php

```diff
@@ -57,22 +57,25 @@ public function __construct(
 		array $attrs = [],
 		Fields|null $siblings = null
 	) {
+		// use the type as fallback for the name
+		$attrs['name'] ??= $type;
+		$attrs['type']   = $type;
+
+		// set the name to lowercase
+		$attrs['name'] = strtolower($attrs['name']);
+
 		if (isset(static::$types[$type]) === false) {
 			throw new InvalidArgumentException(
 				key: 'field.type.missing',
 				data: [
-					'name' => $attrs['name'] ?? '-',
-					'type' => $type
+					'name' => $attrs['name'],
```

Comment:
`Nitpick:` This will produce a (maybe) misleading error message when the name here might already be filled in with the type, but the error message presents it as its actual name.

Reply:
I've reverted it